qiita_ware/util.py

```diff
@@ -117,104 +116,14 @@ def stats_from_df(df):
     out = {}
 
     for column in natsorted(df.columns):
+        if column in {'study_id'}:
```

Comment:
Did this as a way to allow easy extensibility of ignored columns, but can hardcode that for now.